### PR TITLE
clarify how to deal with pre-existing CBOR tags

### LINKED ISSUE #8: what about pre-existing CBOR tags?
We need to define how to deal with CBOR tags that exist independently of a CoAP C-F.

### PR DESCRIPTION
**Pull request by thomas-fossati**
(also `make fix-lint` to remove trailing whitespaces.)

Fix #8

Signed-off-by: Thomas Fossati <thomas.fossati@arm.com>

### CODE REVIEW

**Review by nedmsmith — APPROVED** [review]

File: draft-ftbs-rats-msg-wrap.md [hunk 1]
```diff
@@ -146,7 +151,22 @@ When using CBOR, the value field is serialized as a CBOR bytes string.
...
+### Use of Pre-existing CBOR Tags
+
+If a CBOR tag has been registered in association with a certain RATS
+conceptual message independently of a CoAP Content Format (i.e., it is
+not obtained by applying the TN() transform), it can be readily used as
+an encapsulation without the extra processing described in {{cbor-tag}}.
+
+A consumer can always distinguish tags that have been derived via TN()
+from tags that are not, and therefore apply the right decapsulation on
+receive.
```
Comment (henkbirkholz) [inline draft-ftbs-rats-msg-wrap.md:169]: Maybe a quick 'how' to be on the safest side?

File: draft-ftbs-rats-msg-wrap.md [hunk 2]
```diff
@@ -177,15 +197,15 @@ alongside a corresponding CoAP content format `30001`.  The CBOR tag
...
+there are no further security requirements raised by
 the introduction of this encapsulation.
 
 Changing the encapsulation of a payload by an adversary
-will result in incorrect processing of the encapsulated 
-messages and this will subsequently lead to a processing 
-error. 
+will result in incorrect processing of the encapsulated
+messages and this will subsequently lead to a processing
+error.
```
Comment (henkbirkholz) [inline draft-ftbs-rats-msg-wrap.md:208]: if your process has exactly zero other ways of checking that is

**Review by henkbirkholz — APPROVED** [review]
lgtm

File: draft-ftbs-rats-msg-wrap.md [hunk 3]
```diff
@@ -58,20 +58,24 @@ informative:
 --- abstract
 
 This document defines two encapsulation formats for RATS conceptual
-messages (e.g., evidence, attestation results, endorsements and
+messages (i.e., evidence, attestation results, endorsements and
 reference values.)
 
+The first format uses a CBOR or JSON array with two slots: one for the
+type, another for the value.  The other format wraps the value in a CBOR
+byte string and prepends a CBOR tag to convey the type information.
```
[the remark this answers is not in the document]
Comment (thomas-fossati) [reply draft-ftbs-rats-msg-wrap.md]: not sure why this got duplicated.  resolving a846601

File: draft-ftbs-rats-msg-wrap.md (hunk 1 above)
Comment (thomas-fossati) [reply draft-ftbs-rats-msg-wrap.md]: done in f5e20f2

**Pull request merged**